lib/spack/spack/cmd/create.py

```diff
@@ -58,7 +57,7 @@ class {class_name}({base_class_name}):
 
     # FIXME: Add a proper url for your package's homepage here.
     homepage = "http://www.example.com"
-    url      = "{url}"
+{url}
```

Comment:
call this url_definition or something like that, as `url` makes me think it's still just the URL.